pages/fun_facts.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="../stylesheets/style.css">
+    <link rel="stylesheet" href="../stylesheets/fun_facts.css">
+</head>
+<body>
+    <header>
+        <div id="header">
+            <h1>Fun Facts</h1>
+        </div> 
+        <nav>
+            <ul>
+                <li><a href="../index.html">Home</a></li>
+                <li><a href="education.html">Education</a></li>
+                <li><a href="hobbies.html">Hobbies</a></li>
+            </ul>
+        </nav>
+        <div class="additional_header">
+            <h2>Can you figure out which fun facts are true?</h2>
+        </div>
+            <section class="fun-fact-container">
+                <img src="../images/Adrian-Prado-600x600.png" alt="Hey, Adrian!" width="300" height="200">
+                <h3>Adrian Prado</h3>     
+                <p>Passionate about escape rooms and has mastered the art of outsmarting tricky puzzles and locks.</p>
+                <p>Aspires to visit every U.S. National Park; already checked 15 off the list and counting..</p>
+                <p>Holds dual citizenship: American through his mother and Costa Rican through his father,embracing the best of both cultures.</p>
+                <p>Proud owner of a pet named 404—who, true to their name, is still not found.</p>
+                <p>Can write code so clean, even the bugs get lost</p>
+                <p>Once Won an Argument with a Compiler</p>
```

Comment:
Since these aren't necessarily large body paragraphs, we could change them to list items in an unordered list and just remove the list item styling.